### ISSUE TITLE
Database creation errors

### ISSUE THREAD

**Issue by avimar** [tony-landis/agilebill#2]
I did a git clone and ran the install script. I selected a bunch of plugins and I saw these errors:

ALTER TABLE ab_faq_translate ADD INDEX fulltext_faq (answer, question)
Error (install_sql-3): BLOB/TEXT column 'answer' used in key specification without a key length
ALTER TABLE ab_ticket ADD INDEX fulltext_content (subject, body)
Error (install_sql-3): BLOB/TEXT column 'body' used in key specification without a key length
ALTER TABLE ab_ticket_message ADD INDEX fulltext_message (message)
Error (install_sql-3): BLOB/TEXT column 'message' used in key specification without a key length
ALTER TABLE ab_ticket_attachment ADD INDEX fulltext_content (content)
Error (install_sql-3): BLOB/TEXT column 'content' used in key specification without a key length

**Comment by edude03** [comment]
Yeah I just cloned and ran it, having the same issue. Bunch of SQL errors and no tables were created in the database. 

Running mySQL 5.5.12 on Cent5.6
